docs/user/planner/signup_methods.rst

```diff
@@ -0,0 +1,57 @@
+Signup methods
+==============
+
+To cover different use cases, ephios offers different signup methods. In the following, the provided logon procedures are explained, but additional signup methods can also be integrated as a plugin.
+
+When creating a shift, a signup method has to be selected. This controls how users can sign up for the shift and what happens after signing up. 
+
+Availabe signup methods
+-----------------------
+Instant confirmation
+^^^^^^^^^^^^^^^^^^^^
+
+Shifts with this signup methods offer the user a button with the inscription "Participate". Any user who signs up is immediately confirmed and listed as a participant in the shift.
+
+The shift settings can specify conditions that a user must meet in order to participate. These include a minimum age and a registration deadline. In addition, a minimum and a maximum number of participants can be specified. If the maximum number of participants is reached, no further users can register. Furthermore, required qualifications can be specified. Users must have all the specified qualifications to be able to participate.
+
+In addition, it can be set whether users who have already been confirmed can cancel on their own and whether individually deviating start and end times can be specified when registering.
+
+Request and confirm
+^^^^^^^^^^^^^^^^^^^
+
+Shifts with this registration procedure offer the user a button labeled "Requests". Any user requesting to attend will have their request listed in the disposition. The persons responsible for the event will receive notification of the request. In the disposition they can then confirm or reject the participation. 
```

Comment:
I don't think we should introduce yet another term ("registration procedure").

Reply:
yeah that is due to DeepL. as stated above, I only checked the english version roughly. DeepL uses a lot of different words for the same german word. This is all Anmeldeverfahren in german, but DeepL uses registration procedure, logon procedure, signup procedure etc. in the same translation run. Same was true for thinks like Zahnrad where i had cog, cogwheel and gear in the translation